ray-operator/apis/ray/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -118,6 +118,11 @@ func (in *DashboardStatus) DeepCopy() *DashboardStatus {
 // DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
 func (in *HeadGroupSpec) DeepCopyInto(out *HeadGroupSpec) {
 	*out = *in
+	if in.HeadService != nil {
```

Comment:
This is unrelated to #1105. The generated code is for #1040. We should enhance the CI to detect such issues. The command `make docker-image` will not trigger the local `zz_generated.deepcopy.go` generation. We need to run `make build` to update `zz_generated.deepcopy.go`.

Reply:
Open an issue: #1111